newsrevenuehub/rev-engine
# DEV-2000 | Suppress Sentry alert for disallowed host

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

This issue required a slightly different flow with regard to review apps. Since review apps don't log to Sentry, I pushed the changes from this feature branch to `test`, so that can do UAT there.

#### What's this PR do?

Configures Sentry to ignore DisallowedHost errors


#### Why are we doing this? How does it help us?

Reduce Sentry noise

#### How should this be manually tested?

1. Open [this Sentry issue](https://sentry.io/organizations/news-revenue-hub/issues/3305198505/events/d229f8ff80154b93937150030206f6a2/?project=6046263). Find the "Last seen" value, and take note of that.
2. Visit [https://rev-engine-test.herokuapp.com/](https://rev-engine-test.herokuapp.com/) (this is the disallowed host that was reported on Sentry). You should get a "Bad Request (400)" screen.
3. Refresh the Sentry issue after ~30 seconds. The "Last seen" value should not be "a few seconds ago" but ~what it was before.

#### Have automated unit tests been added?

N/A

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

No

#### Has this been documented? If so, where?

N/A

#### What are the relevant tickets? Add a link to any relevant ones.

[DEV-2000](https://news-revenue-hub.atlassian.net/jira/software/c/projects/DEV/boards/3?modal=detail&selectedIssue=DEV-2000)

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No

#### Are there next steps? If so, what? Have tickets been opened for them?

No
